sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/util/SchemaNoPersistence.java

```diff
@@ -0,0 +1,35 @@
+package io.stargate.sgv2.restsvc.util;
+
+import io.stargate.db.schema.ImmutableKeyspace;
+import io.stargate.db.schema.Keyspace;
+import io.stargate.db.schema.Schema;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Bogus {@link Schema} implementation that we need for Stargate V2 as we do not have actual
+ * Persistence implementation. This means that keyspace and table entities are constructed on-demand
+ * without validation for existence.
+ */
+public class SchemaNoPersistence extends Schema {
+  private static final Keyspace ANONYMOUS = ImmutableKeyspace.builder().name("<anonymous>").build();
+
+  @Override
+  public Set<Keyspace> keyspaces() {
+    return Collections.emptySet();
+  }
+
+  @Override
+  public Keyspace keyspace(String name) {
+    if (name == null) {
+      return ANONYMOUS;
+    }
+    return ImmutableKeyspace.builder().name(name).build();
+  }
+
+  public List<String> keyspaceNames() {
+    return keyspaces().stream().map(k -> k.name()).sorted().collect(Collectors.toList());
```

Comment:
Is this implemented like this as it's expected that the `keyspaces()` could be overriden? Also it feels a bit strange that `keyspaces` and `keyspaceNames` would return empty stuff, but if you reference a keyspace by name in the `keyspaceNames`, then you would get the non-null result?

Reply:
This was copy-pasted from `Schema`: I'll go ahead and simply return empty List instead, makes more sense in this case. Thanks!

Reply:
What about the second point I mentioned?

Reply:
From modeling perspective this is not right, I agree; but at this point this is the way to be able to use Stargate `QueryBuilder`. I am not sure we necessarily should -- there is an alternative from Cassandra Java driver -- but at this point we are still prototyping.
If we decide to try to make this QB work better, be used, need to figure out how to refactor it in a way that does not rely on `Schema`.